docs/building.html

```diff
@@ -440,7 +440,7 @@ <h2>Building the code<a class="headerlink" href="#id1" title="Permalink to this
 <span class="nb">cd</span> tello
 <span class="c1"># Install SLAM here using ``bash -i install_openvslam`` OR run the cmake command below with -DUSE_CMAKE_NOT_SCRIPT</span>
 mkdir -p build <span class="o">&amp;&amp;</span> <span class="nb">cd</span> build
-cmake -DRUN_SLAM ..
+cmake -DRUN_SLAM=ON ..
```

Comment:
I have a different source folder for building the docs; I'll make the change and ensure that you are given credit for spotting the missing line.

Reply:
Thanks!